package-lock.json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Since we're using Bundler, I don't think we need this file 💎

Reply:
Great suggestions! All have been changed!